rfcs/0004-insecure-http/README.md

```diff
@@ -111,9 +111,16 @@ for the required commands, which will be used for specifying the value of `.spec
 > Note: This flag should not be confused with `--insecure-skip-tls-verify` which is meant to skip TLS verification
 > when using an HTTPS connection.
 
+### Proxy
+
+The flag shall also respect all possible proxy configurations. If HTTP requests are blocked, then specifying the
```

Comment:
"override" doesn't seem apt here, since the flag is not really providing another value to override the original value with. 
what about "The flag shall also be applicable to all possible proxy configurations."?